python/tk_multi_loader/dialog.py

```diff
@@ -1180,16 +1201,26 @@ def _add_rdo_list_view_mod(self):
             self.ui.label_2.hide()
             self.ui.thumb_scale.hide()
 
-    def _add_rdo_publish_search(self):
+    def _add_rdo_status_filter(self, hlayout):
+        '''
+        Add a combo box to sort the latest publishes by status
+        Kind of copying off the _add_rdo_status_filter
+        '''
+        filter_label = QtGui.QLabel("Filter by Status")
```

Comment:
Whoops for some reason I thought the status filter beside the search is what we wanted, but I can move it somewhere else. I changed the combobox to adjust to contents and tested it with and without the icons. The only one that doesn't like it is softimage. For some reason the softimage version resizes the combobox to be exactly the size of the text and then doesn't account for the little down arrow... both maya and nuke adjust accordingly. I just added a bunch of spaces afterwards to compensate for the xsi weirdness